moduleroot/Rakefile.erb

```diff
@@ -77,6 +77,15 @@ PuppetLint.configuration.send('<%= option %>')
 if Bundler.rubygems.find_name('github_changelog_generator').any?
   GitHubChangelogGenerator::RakeTask.new :changelog do |config|
     raise "Set CHANGELOG_GITHUB_TOKEN environment variable eg 'export CHANGELOG_GITHUB_TOKEN=valid_token_here'" if Rake.application.top_level_tasks.include? "changelog" and ENV['CHANGELOG_GITHUB_TOKEN'].nil?
+<% if @configs['github_site'] -%>
+    config.github_site = <%= @configs['github_site'].inspect %>
```

Comment:
Without tests this could be _interesting_.  `.inspect` has bitten us before with YAML files.  But as this is a ruby file and inspect generates ruby code, this is _probably_ safe (?)